content/product-engineering/engineering/cloud/devops/continuous_delivery.md

```diff
@@ -0,0 +1,11 @@
+# Sourcegraph Cloud continuous delivery
```

Comment:
Btw this is a duplicate of this page, we should just update the existing page: https://handbook.sourcegraph.com/product-engineering/engineering/deployments

Reply:
For things that apply to engineering as a whole, we should try to centralize what we know so that it's easier to maintain and fine

Reply:
Thanks, I wasn't 100% sure where the documentation should live. I'm happy to merge the two documents. Do you think it belongs on [deployments](https://handbook.sourcegraph.com/product-engineering/engineering/deployments) or should I just add further elaboration to [Deploying to sourcegraph.com](https://handbook.sourcegraph.com/product-engineering/engineering/deployments/playbooks#deploying-to-sourcegraphcom)?

Reply:
If you look at [Deployments basics](https://handbook.sourcegraph.com/product-engineering/engineering/deployments#deployment-basics) it already covers (in a less complete manner) some of the things you've written up here, so I would just update this section

Reply:
[this also needs to be updated](https://handbook.sourcegraph.com/product-engineering/engineering/deployments#deploy-sourcegraph) to remove references of the renovate downstream/dispatch workflows, those are no longer used

Reply:
Thank you @bobheadxi . I've made the changes discussed. If you get another chance to take a look at the latest changes that'd be greatly appreciated!